MidtransCoreKit/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
```

Comment:
could you explain, how to generate this file?

Reply:
you can go to xcode, and go to File → New → File , scroll down a little bit to the Resource section and select App privacy